### ISSUE TITLE
Please insert / mount disc 1(install disc)

### ISSUE THREAD

**Issue by first-coding** [wheybags/EmperorLauncher#8]
i want to install it,but it appear this problem,I like this game and want to paly game with other people.

**Comment by wheybags** [comment]
Did you insert or mount the install cd?

**Comment by first-coding** [comment]
Yes, I inserted the cd

**Comment by wheybags** [comment]
Are you using the English version of the original cds? If you're using a repack or other language version, it might not work.

**Comment by AlexDCIT** [comment]
> Did you insert or mount the install cd?

So what if my pc doesn't have disc drive, so i can only mount discs?

**Comment by wheybags** [comment]
Yes, you can mount an iso of the original english install CD and it should work. Windows even has built in iso mounting support these days
